docs/guides/controller-autoconfiguration.md

```diff
@@ -36,34 +37,53 @@ This allows:
 
 A message will appear at the bottom of the screen showing the download progress and the extraction of the archive.
 
+
 ## Generating a joypad profile
 
 If your joypad is not recognized by RetroArch even after updating the profiles, you can generate a profile from the menu.
 
-
 1. Unplug all the other joypads
-2. Use `Settings` -> `Input` -> `User 1 Binds` -> `User 1 Bind All`. Make sure that your mapping is perfect by testing every button in the menu and in some games.
-3. Use `Settings` -> `Inputs` -> `User 1 Binds` -> `User 1 Save Autoconfig`
-4. The new profile will be saved to your disk.
-5. Use `Settings` -> `Inputs` -> `User 1 Binds` -> `User 1 Bind Default All` to clear manual bindings and rely on the new profile
-6. Unplug your joypad an re-plug it. See if it is auto configured.
+2. For Android, run the [Input descriptors](#android) step first.
+3. Use `Settings` -> `Input` -> `RetroPad Binds` -> `Port 1 Controls` -> `Set All Controls`. Make sure that your mapping is perfect by testing every button in the menu and in some games.
+4. If applicable, then also set the menu button binding in `Settings` -> `Input` -> `Hotkeys` -> `Menu Toggle`
+5. Use `Settings` -> `Input` -> `RetroPad Binds` -> `Port 1 Controls` -> `Save Controller Profile`
+6. The new profile file will be saved to your disk: [Controller profile directory]/[Controller driver]/[Device index].cfg
+7. Use `Settings` -> `Inputs` -> `Port 1 Controls` -> `Reset to Default Controls` to clear manual bindings and rely on the new profile
+8. Unplug your joypad an re-plug it. See if it is auto configured.
 
 
 !!! Warning "Clear manual bindings"
-    It is important to to skip the step of clearing manual bindings after using the `User 1 Save Autoconfig` command. In order to avoid issues using your profile in the future, remember to go to `Settings` -> `Inputs` -> `User 1 Binds` -> `User 1 Bind Default All` to reset the manual settings before completing this process.
+    It is important to to skip the step of clearing manual bindings after using the `Save Controller Profile` command. In order to avoid issues using your profile in the future, remember to go to `Settings` -> `Input` -> `RetroPad Binds` -> `Port 1 Controls` -> `Reset to Default Controls` to reset the manual settings _before_ completing this process.
 
 If you are happy with your profile, you can submit it to RetroArch so that other users benefit:
 
-1. Edit the autoconfig file for your joypad manually to include the input descriptors (please see the **Input Descriptors** section below)
-2. [Submit your profile to our joypad profile repository](https://github.com/libretro/retroarch-joypad-autoconfig).
+1. Edit the autoconfig file for your joypad manually to include the input descriptors (please see the [Input descriptors](#input-descriptors) section below)
+3. [Submit your profile to our joypad profile repository](https://github.com/libretro/retroarch-joypad-autoconfig).
```

Comment:
``` suggestion
2. [Submit your profile to our joypad profile repository](https://github.com/libretro/retroarch-joypad-autoconfig).
```

Reply:
You are right. I added the wrong text before to the link. But i correctet it now in https://github.com/libretro/docs/pull/910/commits/5d8f06e7bc41416c11d6ea14b541ce022358fc15